src/pages/location-message/location-message.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { NavController, ViewController, AlertController, Platform } from 'ionic-angular';
+import { Geolocation } from 'ionic-native';
+import { Location } from 'api/models';
+
+const DEFAULT_ZOOM = 8;
```

Comment:
It's in use inside the specific file, and I don't want it to be defined each time the component runs/created. 
This is not exported, and relevant only to this file...